nntrainer/compiler/flatbuffer_opnode.cpp

```diff
@@ -0,0 +1,93 @@
+// SPDX-License-Identifier: Apache-2.0
+/**
+ * Copyright (C) 2023 DongHak Park <donghak.park@samsung.com>
+ *
+ * @file   flatbuffer_opnode.h
+ * @date   10 February 2023
+ * @brief  NNTrainer flatbuffer opnode
+ * @see	   https://github.com/nnstreamer/nntrainer
+ * @author Donghak Park <donghak.park@samsung.com>
+ * @bug    No known bugs except for NYI items
+ */
+
+#include <flatbuffer_opnode.h>
+
+#include <layer_context.h>
+#include <layer_node.h>
+#include <memory.h>
+
+namespace nntrainer {
+
+FlatBufferOpNode::FlatBufferOpNode() :
+  inputs(),
+  outputs(),
+  weights(),
+  is_input(),
+  is_output(),
+  is_virtual(),
```

Comment:
```suggestion
  is_virtual(false),
```
?? what is default value?

Reply:
[FIX] Thank you for Review